docs/pages/management/guides/fluentd.mdx

```diff
@@ -38,11 +38,17 @@ from Teleport's events API, and forwards them to Fluentd.
   ```
   </TabItem>
 
-  <TabItem label="MacOS">
+  <TabItem label="macOS">
+
   ```code
   $ curl -L -O https://get.gravitational.com/teleport-event-handler-v(=teleport.plugin.version=)-darwin-amd64-bin.tar.gz
-  $ tar -zxvf teleport-event-handler-v(=teleport.plugin.version=)-linux-amd64-bin.tar.gz
+  $ tar -zxvf teleport-event-handler-v(=teleport.plugin.version=)-darwin-amd64-bin.tar.gz
   ```
+
+  We currently only build the event handler plugin for amd64 machines. If your
+  macOS machine uses Apple silicon, you will need to [install
```

Comment:
```suggestion
  macOS machine uses Apple Silicon, you will need to [install
```

Reply:
It looks like Apple's convention is to use "Apple silicon":

- https://www.apple.com/business/mac/pdf/Apple-at-Work-Apple-Silicon-Overview.pdf
- https://support.apple.com/en-us/HT211814